### PR TITLE
Code sintaxices refactored to php8 version

### PR DESCRIPTION
**Pull request by kSolovei**

### CODE REVIEW

File: src/Support/LiqPay.php [hunk 1]
```diff
@@ -21,8 +22,6 @@
  *
  */
 
-namespace DenizTezcan\LiqPay\Support;
```
Comment (deniztezcan) [inline src/Support/LiqPay.php:24]: ???

**Review by deniztezcan — CHANGES_REQUESTED** [review]
See comment

File: src/Support/LiqPay.php (hunk 1 above)
Comment (kSolovei) [reply src/Support/LiqPay.php:24]: See. Will change

Comment (kSolovei) [reply src/Support/LiqPay.php:24]: got it, missed

**Comment by kSolovei** [comment]
I figured out with Guzzle, made post request got status 200, when checked the result of function getStatus() , but I can't get witch data I've send, the data sending body is empty, I figured out the signature. Unfortunately, I did not manage to understand what was going wrong.

**Review by deniztezcan — APPROVED** [review]

**Pull request merged**